site/static/diagrams/system-diagram.dot

```diff
@@ -0,0 +1,165 @@
+digraph {
```

Comment:
Apache licence please 😄

Reply:
Hmm, we don't actually license any of the other .dot files?

Reply:
We probably should 😁

Reply:
Done, PTAL